dev-tools/cmd/module_include_list/module_include_list.go

```diff
@@ -99,19 +99,18 @@ func main() {
 	// Build import paths.
 	var imports []string
 	for _, dir := range dirs {
-		// Skip dirs that have no .go files.
-		goFiles, err := filepath.Glob(filepath.Join(dir, "*.go"))
-		if err != nil {
-			log.Fatal("Failed checking for .go files in package dir: %v", err)
-		}
-		if len(goFiles) == 0 {
```

Comment:
I think this was used as a sanity check with the assumption that anything using this code should find at least one thing to import. Maybe we should add this check after the for loop completes?

Reply:
I removed this check because at the end if there are no go files then `foundInitMethod` is never going to be `true`.